Makefile

```diff
@@ -74,6 +74,9 @@ test:
 				exit 1; \
 			fi; \
 	    	done; \
+	set -ex; for f in $$(pwd)/tests/test_*.sh; do \
+		sh -e $$f; \
+	done
```

Comment:
Why `-e` twice and `-x` only once?

Reply:
The global "-ex" ensures we see what test is run. The sh -e just ensures the individual test fails when it should fail. Note that the test ifile tself adds a "-x/+x" in the test section to make sure that failures are visible. It's a very poor mans testsuite and I would be happy to replace it one day but for now I think it's sufficient.